src/content/advanced/configure-dex-in-your-cluster/index.md

```diff
@@ -216,7 +216,7 @@ data:
 
 __Warning__: With `oidc` connector you might need to add `getUserInfo` in the connector configuration to force a second call to the identity provider in order to get groups. This is required for example by Okta. More info on this can be found in [dexipd/dex#1065](https://github.com/dexidp/dex/issues/1065).
 
-__Note__: In the above snippet you have to replace the `<CLUSTERID>` variable and add the Kubernetes Certificate Authority to ensure Dex can trust the API endpoint. Finally you have to use a connector. Here we show examples for Keycloak, Active Directory, and GitHub.
+__Note__: In the above snippet you have to replace the `<CLUSTERID>` variable and select a connector. Here we show examples for Keycloak, Active Directory, and GitHub.
```

Comment:
We dont need to add the CA anymore

Reply:
Why? Is it not needed any more for k8s-dex-authenticator? See https://docs.giantswarm.io/ui-api/management-api/authentication/login-utility-results.png

Reply:
it is taken from cluster configmap values

Reply:
so customer does not need to provide it when they use `customer` connector

Reply:
Cool, thank you!